projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -48,6 +49,18 @@ import { DOCUMENT, NgIf } from '@angular/common';
  * </igx-drop-down>
  * ```
  */
+
+const defaultOverlaySettings: OverlaySettings = {
+    closeOnOutsideClick: true,
+    modal: false,
+    positionStrategy: new ConnectedPositioningStrategy({
```

Comment:
you can simplify this by call:
```ts
positionStrategy: new ConnectedPositioningStrategy()
```
No need to provide `positionSettings` as these are default values for this strategy.